### DISCUSSION TITLE
📔【操作系统】进程与线程

### DISCUSSION THREAD

**Discussion by imageslr** [imageslr/blog#72] [Announcements]
https://imageslr.github.io/2020/07/07/process-thread.html 

 进程是一个拥有资源和执行任务的单元体。进程拥有的资源包括：内存空间中的代码、数据等；I/O 资源；文件；处理机等。

**Comment by Lehirt** [comment]
总结的非常棒！如果博主能更新一下Linux关于进程和线程的知识就更好了！

**Comment by imageslr** [comment]
请问是关于哪方面的？源码层面？有哪些相关问题可以参考。

**Comment by Lehirt** [comment]
> @imageslr
> 请问是关于哪方面的？源码层面？有哪些相关问题可以参考。

Linux中模糊了线程的概念，线程和进程都是task_struck结构，那么进程的fork开销和切换开销究竟比线程快在哪里？或者说进程的fork和切换有没有一个比较具体的过程？https://www.zhihu.com/question/19903801   知乎上的这个问题本身是不是就有问题？
因为对Linux没有具体学习过，所以这方面可能会比较模糊，更希望能看到一篇具体的介绍。

**Comment by HuXia7157** [comment]
是不是少了一块线程池的总结呀

**Comment by imageslr** [comment]
@HuXia7157 是没有，这部分资源比较多，暂时就不准备总结了。
